src/components/Button/Button.js

```diff
@@ -1,8 +1,21 @@
 import React from 'react';
 
 const button = (props) => {
+
+  const handleClick = (buttonName) => {
```

Comment:
## [ IMPORTANT CHANGE ]
In relation to my comment in `ButtonPanel.js` about `clickHandler` passing, this right here is redundant of what we did there. But this is indeed the better place to put this kind of logic. Although I find it odd that we are passing `buttonName` as parameter but not actually using it (??). Also we can do the same object deconstruction as I mentioned before (this is why we need the linter so badly, and of course have a good understanding of ES6+).

Why don't we just:

```javascript
 const { name, clickHandler } = props; // you should also include other props in this deconstruction
 const handleClick = () => clickHandler(name);
 
}
```

Reply:
It won't let me implement this either :neutral_face: